pkgs/by-name/mo/monophony/package.nix

```
@@ -0,0 +1,77 @@
+{ lib
+, fetchFromGitLab
+, python3Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
+, libadwaita
+, libsoup_3
+, glib-networking
+,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "monophony";
+  version = "2.2.2";
+  format = "other";
+
+  sourceRoot = "source/source";
+  src = fetchFromGitLab {
+    owner = "zehkira";
+    repo = "monophony";
+    rev = "v${version}";
+    hash = "sha256-vAX9ac706fKVYW1jjgUJ2GvGRWeytYfgf7Lal+gEm04=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    brotli
+    gst-python
+    mutagen
+    ordered-set
+    pycryptodomex
+    pygobject3
+    requests
+    websockets
+    ytmusicapi
+    mpris-server
+  ];
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook4
+    python3Packages.nuitka
+  ];
+
+  buildInputs =
+    [
+      libadwaita
+      libsoup_3
+      glib-networking
```

```suggestion
      # needed for gstreamer https
      libsoup_3
      glib-networking
```

indicate that these are needed for https to work as removing them will not cause the build to break and is non-obvious why they are needed.

---

Okay, will do.

---

Done